hier_config/hc_child.py

```diff
@@ -569,7 +572,7 @@ def _config_to_get_to_left(self, target, delta):
                 deleted.negate()
                 if self_child.children:
                     deleted.comments.add(
-                        f"removes {len(self_child.children_dict) + 1} lines")
+                        "removes {} lines".format(len(self_child.children_dict)))
```

Comment:
The `len(self_child.children_dict) + 1` counts the negation of the child and all of its children.  We will want to keep the `+ 1`.